src/VirtualPage.php

```diff
@@ -99,61 +99,61 @@ public function getJsUrl($mode = 'callback')
      */
     public function getHtml()
     {
-        if ($this->cb->isTriggered() && !$this->cb->canTerminate()) {
+        if (!$this->cb->isTriggered()) {
```

Comment:
@ibelar is this change correct? Previously not checked if `$this->cb->canTerminate()` was true. Now if not triggered, output is never shown.

Reply:
not sure what you mean but it should be like this: https://github.com/atk4/ui/blob/2b03fd9808b05b3b320fefa4ba27ad56ee1d9fd6/src/VirtualPage.php#L102

If it is triggered but not terminated, it should return parent::getHtml()

Reply:
see https://github.com/atk4/ui/blob/2b03fd9808b05b3b320fefa4ba27ad56ee1d9fd6/src/VirtualPage.php#L106 cond, the code did:
```
if ($this->cb->isTriggered() && !$this->cb->canTerminate()) {
    return parent::getHtml();
}

if ($this->cb->canTerminate()) {
    ... return string
}
```

my question is, if the 2nd condition is complete